### PR TITLE
[Housekeeping] Added pending changes in Android AlertManager

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

From a conversation with @PureWeen, this PR include pending changes in the Android `AlertManager`.

### CODE REVIEW

**Comment by jsuarezruiz** [comment]
@PureWeen  Could you take a look to this one?

File: src/Controls/src/Core/Platform/AlertManager/AlertManager.Android.cs [hunk 1]
```diff
@@ -94,6 +94,12 @@ public void ResetBusyCount()
 
 			void OnPageBusy(IView sender, bool enabled)
 			{
+				// Verify that the page making the request is part of this activity 
+				if (!PageIsInThisContext(sender))
```
Comment (PureWeen) [inline src/Controls/src/Core/Platform/AlertManager/AlertManager.Android.cs:98]: Can we add some tests for this one? 

You should be able to use `CreateHandlerAndAddToWindow` to open up multiple windows and set `IsBusy`

It's not completely accurate on android but it is on `Windows` and will be eventually accurate once we have these tests opening up new `Activities`

Comment (jsuarezruiz) [reply src/Controls/src/Core/Platform/AlertManager/AlertManager.Android.cs:98]: Already there is a Device test like that here https://github.com/dotnet/maui/blob/4ac9f5bfacb9cbbac505615983b1538f5a448277/src/Controls/tests/DeviceTests/Elements/Page/PageTests.cs#L86

**Comment by mattleibow** [comment]
"pending changes"? Is this from Xamarin.Forms?

**Comment by jsuarezruiz** [comment]
> "pending changes"? Is this from Xamarin.Forms?

This validation exists in Xamarin.Forms. Is pending on .NET MAUI but is not from a recent change in Forms.

**Review by rmarinho — APPROVED** [review]

**Comment by mattleibow** [comment]
My bewPR replaces the whole system, so we can merge this now to reduce bad things and then those additional test will help catch anything I break.

**Review by mandel-macaque — APPROVED** [review]

**Pull request merged**